### PR TITLE
 CB-9158 - InAppBrowser zoomControls are always set to true

### PR DESCRIPTION
**Pull request by syndbg**
Also minor whitespaces fixes.

### CODE REVIEW

File: src/android/InAppBrowser.java [hunk 1]
```diff
@@ -656,7 +647,7 @@ public void onClick(View v) {
                 WebSettings settings = inAppWebView.getSettings();
                 settings.setJavaScriptEnabled(true);
                 settings.setJavaScriptCanOpenWindowsAutomatically(true);
-                settings.setBuiltInZoomControls(getShowZoomControls());
+                settings.setBuiltInZoomControls(showZoomControls);
```
Comment (syndbg) [inline src/android/InAppBrowser.java]: here's the issue

**Comment by infil00p** [comment]
Why does this pull request touch iOS?  Can you make it so that it's only Android?

**Comment by purplecabbage** [comment]
Numerous issues here .. removing whitespace from the diff reveals only minor changes.
The changes however do not address the issue as reported in JIRA.

**Comment by syndbg** [comment]
The white spaces are automatically stripped by my text editor.

**Comment by syndbg** [comment]
@purplecabbage Good catch. In the JIRA ticket, unfortunately I've pasted the wrong option parameter, `location`, when in fact I meant `zoom`.

This PR fixes `zoom=true` or `zoom=false` not making a difference in the android app.

Can we agree on the fact that fixing (in my opinion) non-formatted code, even in the iOS section of the plugin, should not be a blocker to merge this PR.

**Comment by purplecabbage** [comment]
It is a blocker typically.  Modifications should be restricted to resolving the issue at hand, or at the bare minimum whitespace/formatting changes should be in their own commit.
Reviewing pull requests is enough work without the overhead of having to manually sort through what is relevant.

**Comment by infil00p** [comment]
Agree with @purplecabbage, the iOS whitespace stripping is a blocker.

**Comment by syndbg** [comment]
Alright. Sorry for the delays guys.

Rebased latest upstream master, didn't commit the iOS file and updated the commit message to make more sense.

**Comment by purplecabbage** [comment]
I have tested the change, and verified it. So it is now merged.
Thanks for the contribution @syndbg 
Some great next steps would be to document the feature (under android quirks) and to add a manual test to the list in the /tests/tests.js file.

**Pull request merged**